src/main/java/edu/kis/powp/jobs2d/drivers/command/OperateToCommand.java

```diff
@@ -0,0 +1,19 @@
+package edu.kis.powp.jobs2d.drivers.command;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class OperateToCommand implements DriverCommand {
+    private Job2dDriver driver;
+    private int x;
+    private int y;
+
+    public OperateToCommand(Job2dDriver driver, int x, int y) {
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).